CHANGELOG.md

```diff
@@ -71,6 +71,7 @@
 ## Bug fixes
 
 - On slow connections or in rare corner cases, it was possible that the same record got saved to the database twice. This was caused by a bug in how the unique technical identifier we generate were stored as FHIR. The backend now ensures every record is submitted only once. [#7477](https://github.com/opencrvs/opencrvs-core/issues/7477)
+- Fix place of event of address line 1, 2 & 3 not being updated properly. [[#7531](https://github.com/opencrvs/opencrvs-core/issues/7531)]
```

Comment:
@Nil20 can you make this a little more descriptive please? Summarise shortly in what part of the application the issue happens and how. Something along the lines of:

>When making a correction and changing the address for the place of event, the address lines were incorrectly duplicated (ticket number)

This might not be correct. You know the bug better :)